Dockerfile

```diff
@@ -84,6 +85,7 @@ RUN ./configure \
 	--enable-subnet \
 	--with-pthreads \
 	--with-libevent \
+    --with-libhiredis \
```

Comment:
nitpick, but would you mind fixing the whitespace on these lines? it looks like the other rows are using tabs

Reply:
Sure!